nuxt.config.js

```diff
@@ -78,4 +76,7 @@ export default {
     }
   },
   ignore: ['**/*.test.*', '**/*.spec.*'],
+  publicRuntimeConfig: {
+    baseURL: process.env.BASE_URL || 'http://localhost:8080/api',
+  },
```

Comment:
From Nuxt v2.13 they use runtime config.

Reply:
I know its runtime config, but please make sure axios is configured with this variable

Reply:
OK